packages/mui-styles/src/makeStyles/makeStyles.js

```diff
@@ -164,7 +164,7 @@ function useSynchronousEffect(func, values) {
   let output;
 
   // Store "generation" key. Just returns a new object every time
-  const currentKey = React.useMemo(() => ({}), values); // eslint-disable-line react-hooks/exhaustive-deps
+  const currentKey = React.useMemo(() => ({}), [values]); // eslint-disable-line react-hooks/exhaustive-deps
```

Comment:
Why this change? Tests for `makeStyles` are failing probably because of this change.

Reply:
Was unaware this was causing tests to fail, change was implemented to fix ESLint error 'expected dependency list for useMemo to be an array literal'. In React documentation useMemo's dependency list is also given as an array so I was under the impression that this always must be the case, apologies if I got it wrong. If this isn't causing any errors I can happily revert and silence the ESLint error instead.

Reply:
If you see how `useSynchronousEffect` is used, the `values` parameter is already an array. Can you revert this change and silence the ESLint error?

Reply:
I see now, thanks for the advice, this has now been reverted and silenced.